src/fusiontensor/fusiontensor.jl

```diff
@@ -209,6 +210,30 @@ function FusionTensor(
   return FusionTensor(x, tuplemortar((codomain_legs, domain_legs)))
 end
 
+# specific constructors
+Base.zeros(::Type{T}, fta::FusionTensorAxes) where {T} = FusionTensor(T, fta)
```

Comment:
```suggestion
function Base.zeros(::Type{T}, fta::FusionTensorAxes) where {T}
  ft = FusionTensor{T}(undef, fta)
  fill!(ft, zero(T)
  return ft
end
```
I didn't notice this in previous PRs, but I think we should define the standard constructor for `FusionTensor` as `FusionTensor{T}(undef, fta)` which is filled with undefined data.